src/script/script.h

```diff
@@ -11,12 +11,15 @@
 #include <assert.h>
 #include <climits>
 #include <limits>
+#include <map>
 #include <stdexcept>
 #include <stdint.h>
 #include <string.h>
 #include <string>
 #include <vector>
 
+#include <boost/assign/list_of.hpp>
```

Comment:
Is this actually used in the .h ? If not, why not move it to the cpp or wherever it is used?